### DISCUSSION TITLE
_reader.capture problems in Blazor ServerSide

### DISCUSSION THREAD

**Discussion by javidepuzol** [sabitertan/BlazorBarcodeScanner#31] [General]
good morning.
I am trying to use the component on the Blazor server side.

I have reproduced your example code and everything works great.

But I'm having problems when I try to take a snapshot via _reader.Capture.

I get the following code:

blazor.server.js:1 [2022-02-22T08:05:05.045Z] Error: Connection disconnected with error 'Error: Server returned an error on close: Connection closed with an error.'.
e.log @ blazor.server.js:1
e.stopConnection @ blazor.server.js:1
transport.onclose @ blazor.server.js:1
e.close @ blazor.server.js:1
e.stop @ blazor.server.js:1
(anónimo) @ blazor.server.js:1
(anónimo) @ blazor.server.js:1
(anónimo) @ blazor.server.js:1
a @ blazor.server.js:1
Promise.then (asíncrono)
c @ blazor.server.js:1
(anónimo) @ blazor.server.js:1
G @ blazor.server.js:1
e.stopInternal @ blazor.server.js:1
(anónimo) @ blazor.server.js:1
(anónimo) @ blazor.server.js:1
(anónimo) @ blazor.server.js:1
(anónimo) @ blazor.server.js:1
G @ blazor.server.js:1
e.stop @ blazor.server.js:1
e.processIncomingData @ blazor.server.js:1
connection.onreceive @ blazor.server.js:1
i.onmessage @ blazor.server.js:1
102blazor.server.js:1 Uncaught (in promise) Error: Cannot send data if the connection is not in the 'Connected' State.
    at e.send (blazor.server.js:1:51662)
    at e.sendMessage (blazor.server.js:1:22519)
    at e.sendWithProtocol (blazor.server.js:1:22581)
    at e.send (blazor.server.js:1:22772)
    at Object.beginInvokeDotNetFromJS (blazor.server.js:10:49105)
    at p (blazor.server.js:1:68979)
    at Object.e.invokeMethodAsync (blazor.server.js:1:69816)
    at BlazorBarcodeScanner.js:115:24
    at r (zxingjs.index.min.js:16:34728)

I guess I'm doing something wrong.

Thanks for everything

**Comment by kervingomez** [comment]
Same problem here, seems this does not work on Blazor Server
